src/index.ts

```diff
@@ -46,15 +42,4 @@ async function main(config: any): Promise<FastifyInstance | undefined> {
   await fastify.listen({ port: +(env['PORT'] ?? 80), host: '::' })
 
   return fastify
-}
-
-['SIGTERM', 'SIGINT'].forEach(signal => {
```

Comment:
Why remove this?

Reply:
this is moved into index.ts file in order to enable unit testing